src/peft/tuners/boft/layer.py

```diff
@@ -78,6 +77,8 @@ def get_fbd_cuda():
     if _FBD_CUDA is not None:
         return _FBD_CUDA
 
+    from torch.utils.cpp_extension import load
```

Comment:
Let's add a comment: `# This import initializes cuda context and should thus be local, see issue 1877`